src/pca.jl

```diff
@@ -47,11 +47,27 @@ transform(M::PCA{T}, x::AbstractVecOrMat{T}) where {T<:Real} = transpose(M.proj)
 reconstruct(M::PCA{T}, y::AbstractVecOrMat{T}) where {T<:Real} = decentralize(M.proj * y, M.mean)
 
 ## show & dump
-
 function show(io::IO, M::PCA)
     print(io, "PCA(indim = $(indim(M)), outdim = $(outdim(M)), principalratio = $(principalratio(M)))")
 end
 
+function show(io::IO, ::MIME"text/plain", M::PCA)
+    print(io, "PCA(indim = $(indim(M)), outdim = $(outdim(M)), principalratio = $(principalratio(M)))")
+    ldgs = projection(M) * diagm(0 => sqrt.(M.prinvars))
+    rot = diag(ldgs' * ldgs)
+    ldgs = ldgs[:,sortperm(rot, rev=true)]
+    ldgs_signs = sign.(sum(ldgs, dims=1))
+    ldgs_signs[ldgs_signs .== 0] .= 1
+    ldgs = ldgs * diagm(0 => ldgs_signs[:])
+    print(io, "\n\nPattern matrix\n")
```

Comment:
```suggestion
    print(io, "\n\nPattern matrix:\n")
```
(for consistency with below)

Also, better be specific about what this matrix is: psych says "standardized loadings".